pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn

```diff
@@ -906,6 +918,42 @@ public OnPlayerCommandText(playerid, cmdtext[]) {
         return 1;
     }
 
+    if (strcmp(cmd, "/vips", true) == 0) {
+        SendClientMessage(playerid, COLOR_ORANGE, "List of VIPs online in Las Venturas Playground");
+
+        new message[128], vipCount = 0, playerLevel[30];
+        for (new player = 0; player <= PlayerManager->highestPlayerId(); player++) {
+            if (!Player(player)->isConnected()) continue;
+            if (!Player(player)->isVip()) continue;
+            if (IsPlayerAdmin(player)) continue;
+
+            
+            if (Player(player)->isVip())
+                format(playerLevel, sizeof(playerLevel), "VIP");
+            if (Player(player)->isDeveloper())
+                format(playerLevel, sizeof(playerLevel), "VIP / Developer");          
```

Comment:
A Developer doesn't always have to be a VIP. I believe showing "Developer" would be enough